server/routes/github_oauth.js

```diff
@@ -408,9 +421,26 @@ router.post("/projects/add_project/", async (req, res) => {
   // TODO: atomic transaction
   // TODO: safeguard and return error messages
   await addNewProject(auth_token, req.body['position'], req.body['experience_lvl'], new_project).catch(e => console.error(e));
-  return res.send({ "result": "success" });
+  return res.send({
+    "result": "success"
+  });
 });
 
+//Positions method
+async function getPositions() {
+  let SQL = 'SELECT * FROM Positions;';
+  return client.query(SQL)
+    .then(result => {
+      console.log(result);
+    })
+    .catch(err => {
+      throw err;
+    });
+}
 
+router.get("/getPositions", (req, res) => {
+  const positions = getPositions(res);
```

Comment:
Let's check if the user is authenticated first here. You can see an example of that from other APIs in this file.